st/protocol

```diff
@@ -58,6 +58,10 @@ popd
 echo "Comparing '$proto_dev' and '$proto_current' files"
 $diff_cmd $proto_dev $proto_current
 exit_code=$?
+if [ $exit_code -ne 0 ]; then
+    echo -n "37protocol ST may fail if there are some local changes done to"
```

Comment:
s/echo -n/echo

"-n" option is not required for echo here.

Reply:
No it will be required as I don't want to print it in 2 lines

Reply:
Okay.